src/lando/main/scm/git.py

```diff
@@ -115,14 +116,20 @@ def push(
                 if token:
                     push_path = f"https://git:{token}@github.com/{match['owner']}/{match['repo']}"
 
-        command += [push_path]
+        common_args += [push_path]
 
         if not push_target:
             push_target = self.default_branch
 
-        command += [f"HEAD:{push_target}"]
+        push_command = [*common_args, f"HEAD:{push_target}"]
 
-        self._git_run(*command, cwd=self.path)
+        self._git_run(*push_command, cwd=self.path)
+
+        # If any tags were passed, ensure they are pushed.
+        if tags:
+            for tag in tags:
+                push_tags_command = [*common_args, f"refs/tags/{tag}"]
+                self._git_run(*push_tags_command, cwd=self.path)
```

Comment:
I think we can push the tags at the same time 
![image](https://github.com/user-attachments/assets/e6f3ccf2-342f-414a-8025-f42a38192c57)